docs/integration-tests.md

```diff
@@ -0,0 +1,27 @@
+---
+id: integration-tests
+title: Integration Test Docs
+slug: /integration-tests
+---
+
+# Integraiton Tests
+
+## About
+
+The integration test suite is an end-to-end test suite that uses Selenium and Firefox to verify nimbus functions properly for users.
+
+## Nimbus Tests
+### Getting Started
+
+1. Create a python virtual environment
+2. Copy `.env.intergation-tests` file to `.env`.
+3. Run the `make up_prod_detached` command.
+4. Run the following command to run the integration tests: `make integration_test_nimbus PYTEST_ARGS="$PYTEST_ARGS"`
```

Comment:
I think `PYTEST_ARGS="$PYTEST_ARGS"` is redundant here since `$PYTEST_ARGS` will evaluate to nothing in the container and just default to running the entire suite.

Reply:
Okay I can add a line to set that variable if someone needs to